tools/src/main/python/readconfig.py

```diff
@@ -26,6 +26,11 @@
 import yaml
 import sys
 
+try:
+    from json.decoder import JSONDecodeError
+except ImportError:
+    JSONDecodeError = ValueError
```

Comment:
I'd just add a note that this is because python 3.4 for anybody in the future looking into this

Reply:
Maybe Python should introduce `twenty` package that would bridge 3.4 -> 3.5 (just like there is the `six` package for transition from Python 2 to 3)...